lib/chef/context.rb

```diff
@@ -0,0 +1,65 @@
+# freeze_string_literal: true
+#
+# Copyright:: Copyright (c) Chef Software Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+class Chef
+  class Context
+    class << self
+      COMMON_KEY = "2f3b66cbbafa2d326b2856bccc4c8ebe".freeze
+      FILE_NAME = "c769508738d671db424b7442".freeze
```

Comment:
Can these be handled in a different way? They seem like magic values or keys.

Reply:
Done.